apps/studio/prisma/generated/generatedEnums.ts

```diff
@@ -1,6 +1,6 @@
 export const RoleType = {
-  Admin: "Admin",
-  Editor: "Editor",
-  Publisher: "Publisher",
-} as const
-export type RoleType = (typeof RoleType)[keyof typeof RoleType]
+    Admin: "Admin",
+    Editor: "Editor",
+    Publisher: "Publisher"
+} as const;
```

Comment:
comment: seems like the formatter changed in this whole PR, or somehow your vscode instance got different linter from the rest. I'll review ignoring whitespace, but should be fixed by my future PR when I consolidate the linting and config for the monorepo